chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -568,30 +568,37 @@ func (b *BitcoindNotifier) confDetailsManually(txid *chainhash.Hash,
 // transactions included this block will processed to either send notifications
 // now or after numConfirmations confs.
 func (b *BitcoindNotifier) handleBlockConnected(block chainntnfs.BlockEpoch) error {
+	// First, we'll fetch the raw block as we'll need to gather all the
+	// transactions to determine whether any are relevant to our registered
+	// clients.
 	rawBlock, err := b.chainConn.GetBlock(block.Hash)
 	if err != nil {
 		return fmt.Errorf("unable to get block: %v", err)
 	}
-
 	txns := btcutil.NewBlock(rawBlock).Transactions()
+
+	// We'll then extend the txNotifier's height with the information of
+	// this new block, which will handle all of the notification logic for
+	// us.
 	err = b.txNotifier.ConnectTip(
-		block.Hash, uint32(block.Height), txns)
+		block.Hash, uint32(block.Height), txns, false,
```

Comment:
If this will always be false now, I would remove the `notifyHeight` call from `ConenctTip` instead.

Reply:
Agreed, fixed.